bin/child-npm-projects

```diff
@@ -1,35 +0,0 @@
-#! /usr/bin/env bash
```

Comment:
This whole file was never that useful anyways.  We can safely assume for now that all "shareable" npm projects exist in the `etna/packages` directory and be done with inferring it with logic like this.